capi/Cargo.toml

```diff
@@ -5,7 +5,7 @@ authors = ["Christopher Serr <christopher.serr@gmail.com>"]
 edition = "2018"
 
 [dependencies]
-livesplit-core = { path = "..", default-features = false, features = ["std"] }
+livesplit-core = { path = "..", default-features = false, features = ["std", "auto-splitting"] }
```

Comment:
This is the reason why I didn't pull it in yet. The problem is that unconditionally depending on auto splitting will break for example the web version. So this needs to be an optional feature. However at least atm we don't support features in the binding generator. So either the C API needs to stub out the implementation or we need to improve our binding generator.

Reply:
I've added the feature attribute to everything now, but I still need to work on improving the bindings generator.

That might take me a while since I'm not familiar at all with `syn` and binding generator's code is a bit messy.